### PR TITLE
fix(arc-1736): use the blade manager to show the material requests

### PR DESCRIPTION
**Pull request by bertyhell**
https://meemoo.atlassian.net/browse/ARC-1736

before:
image

after:
image

### CODE REVIEW

File: src/modules/navigation/components/MaterialRequestCenter/MaterialRequestCenterButton.tsx [hunk 1]
```diff
@@ -31,7 +32,9 @@ const MaterialRequestCenterButton: FC = () => {
 	const showMaterialRequestCenter = useSelector(selectShowMaterialRequestCenter);
 
 	const onButtonClick = () => {
+		console.log({ showMaterialRequestCenter });
```
Comment (SHDBramVandenbussche) [inline src/modules/navigation/components/MaterialRequestCenter/MaterialRequestCenterButton.tsx:35]: Remove console log or does it have a purpose ?

File: src/modules/navigation/components/MaterialRequestCenter/MaterialRequestCenterButton.tsx [hunk 2]
```diff
@@ -65,6 +68,7 @@ const MaterialRequestCenterButton: FC = () => {
 		};
 	}, []);
 
+	console.log({ showMaterialRequestCenter });
```
Comment (SHDBramVandenbussche) [inline src/modules/navigation/components/MaterialRequestCenter/MaterialRequestCenterButton.tsx:71]: Same as previous comment

File: src/modules/navigation/components/MaterialRequestCenterBlade/MaterialRequestCenterBlade.tsx [hunk 3]
```diff
@@ -31,11 +37,29 @@ const MaterialRequestCenterBlade: FC<MaterialRequestCenterBladeProps> = ({ isOpe
...
+	const getCurrentLayer = () => {
+		console.log('get current layer: ', { isOpen, activeBlade });
+		if (!isOpen) {
+			return 0;
+		}
+		switch (activeBlade) {
+			case MaterialRequestBladeId.Overview:
+				return 1;
+
+			case MaterialRequestBladeId.EditMaterialRequest:
```
Comment (SHDBramVandenbussche) [inline src/modules/navigation/components/MaterialRequestCenterBlade/MaterialRequestCenterBlade.tsx:55]: Missing case execution?

File: src/modules/navigation/components/PersonalInfoBlade/PersonalInfoBlade.tsx [hunk 4]
```diff
@@ -92,8 +99,13 @@ const PersonalInfoBlade: FC<PersonalInfoBladeBladeProps> = ({ isOpen, onClose, p
 				</h2>
 			)}
 			footer={isOpen && renderFooter()}
-			onClose={onClose}
+			onClose={() => {
```
Comment (SHDBramVandenbussche) [inline src/modules/navigation/components/PersonalInfoBlade/PersonalInfoBlade.tsx:102]: Revert back to `onClose={onClose}`

File: src/modules/navigation/components/MaterialRequestCenterBlade/MaterialRequestCenterBlade.tsx (hunk 3 above)
Comment (bertyhell) [reply src/modules/navigation/components/MaterialRequestCenterBlade/MaterialRequestCenterBlade.tsx:55]: no this is a fall through case
the same thing has to happen for both cases

**Review by SHDBramVandenbussche — APPROVED** [review]

**Pull request merged**